TrainModel.ipynb

```diff
@@ -45,18 +43,97 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = pd.read_csv(\"csd_2020_renamed.csv\")"
+    "data = pd.read_csv(\"csd_2021.csv\")\n",
+    "display(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ects = pd.read_csv(\"subjects_ects.csv\")\n",
+    "ects_dict = ects.set_index('Subjects')['Ects'].to_dict()\n",
+    "#display(ects_dict)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "is_executing": false
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#data.describe()\n",
+    "#data.info()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "\n",
-    "### Categorical Values Handling - Function ###\n"
-   ],
+    "#data['prior_courses'] = number_of_prior_courses\n",
+    "#data['ects_needed'] = ects_needed\n",
+    "#data['ECTS'] = ects_list\n",
+    "#data['GPA'] = gpa_list\n",
+    "#display(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Rename Columns ###"
+   ]
+  },
+  {
+   "cell_type": "code",
    "execution_count": null,
-   "outputs": []
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change Courses Names\n",
```

Comment:
You can find those function in `Parser.ipynb` too. The final csv file, that we extract from the parser, has already the correct names.